ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -666,26 +666,30 @@ def run_ocs_upgrade(operation=None, *operation_args, **operation_kwargs):
 
     # update external secrets
     if config.DEPLOYMENT["external_mode"]:
-        external_cluster.update_permission_caps()
-        external_cluster.get_external_cluster_details()
-
-        # update the external cluster details in secrets
-        log.info("updating external cluster secret")
-        external_cluster_details = NamedTemporaryFile(
-            mode="w+",
-            prefix="external-cluster-details-",
-            delete=False,
-        )
-        with open(external_cluster_details.name, "w") as fd:
-            decoded_external_cluster_details = decode(
-                config.EXTERNAL_MODE["external_cluster_details"]
+        upgrade_version = version.get_semantic_version(upgrade_version, True)
+        if upgrade_version >= version.VERSION_4_10:
+            external_cluster.update_permission_caps()
+        else:
```

Comment:
Hello @parth-gr , if the change with caps will be backported to 4.8 and 4.9, do we need to do change here to pass the user to the script or it will not be needed?